src/middlewares.rs

```diff
@@ -0,0 +1,40 @@
+use actix_web::middleware::{Middleware, Started};
+use actix_web::HttpRequest;
+use actix_web::Result;
```

Comment:
do we prefer this to these three lines of code?
```suggestion
use actix_web::{middleware::{Middleware, Started}, HttpRequest, Result};
```